File: Upcoder/Other/Đếm số lẻ/LTCB-46.cs
using System;
namespace Đếmsốlẻ
{
    public class Program
    {
        static void Main(string[]args)
        {
            int n = int.Parse(Console.ReadLine());
            int temp = 0, count = 0;
            while(n != 0)
            {
                temp = n % 10;
                if(temp % 2 != 0)
                    count += 1;
                n /= 10;
            }
            Console.Write(count);
        }
    }
}

File: Upcoder/Other/Đếm số lẻ/LTCB-46.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
    int n;
    cin >> n;
    int temp, count = 0;
    while(n != 0)
    {
        temp = n % 10;
        if(temp % 2 != 0)
            count += 1;
        n /= 10;
    }
    cout << count;
}